src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx

```diff
@@ -160,6 +160,12 @@ interface ParaTimeSelectorProps extends ParaTimeSelectorBaseProps {
 
 const localStore = storage()
 
+// This is a special layer used to indicate that we should zoom out,
+// and see the whole universe.
+export const UniverseLayer = 'Universe'
+
+export type SelectorLayer = Layer | typeof UniverseLayer
```

Comment:
Would be nice to exclude `pontusx` here, as it is not shown on the graph. And it probably won't be.

Reply:
Why do we need to exclude it? What happens with layers that are not excluded explicitly, but we don't have specific zoom areas defined for them? Nothing, as far as I tell....

Reply:
Nothing, as they can not be clicked.

Reply:
Yeah anyway, I think I'll just explicitly list list the ones that we want, instead of excluding stuff.

Reply:
Works for me.

Reply:
OK, now using:

```
export type SelectorArea =
  | typeof UniverseArea
  | typeof Layer.consensus
  | typeof Layer.cipher
  | typeof Layer.emerald
  | typeof Layer.sapphire
```